Test/FMI3/fmi3_import_default_experiment_test.cpp

```diff
@@ -58,12 +61,20 @@ TEST_CASE("Retrieving default experiment values; values undefined") {
     REQUIRE(fmi3_import_get_default_experiment_tolerance(xml) == 1e-4);
     REQUIRE(fmi3_import_get_default_experiment_step_size(xml) == 1e-2);
 
-    fmi3_import_free(xml);
+    // These will give warnings, since attributes are not set; TODO: Is this the intended behavior?
+    REQUIRE(fmi3_testutil_log_contains(tfmu, "fmi3_xml_get_default_experiment_start: returning default value, since no attribute was defined in modelDescription"));
```

Comment:
These are raised when getting the value despite has_ATTR being false.

Can re-evaluate if we want this warning in the future.